app/src/main/java/org/mozilla/fenix/crashes/CrashContentInteractor.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.crashes
+
+import android.util.Log
+import androidx.annotation.VisibleForTesting
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.Components
+import org.mozilla.fenix.components.appstate.AppAction
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Handles all user interactions with a [CrashContentView].
+ */
+class CrashContentInteractor(
```

Comment:
Can we have a commit within the PR that simply renames `CrashReporterController` to `CrashContentInteractor`? This is mostly for preserving the git history of the file so it doesn't appear as a totally new file. So, only the relevant changes in the file needs to be reviewed instead of the whole file.

Reply:
Will try to accommodate this.

Reply:
Looks like we need a separate commit for each renamed file.
Tried to manually merge the diffs but doesn't seem manageable for bigger diffs with multiple usages of the renamings.